pkg/driverbuilder/local.go

```diff
@@ -69,10 +69,21 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 		if err == nil {
 			slog.Info("Trying automatic kernel headers download.")
 			kernelDownloadScript, err := builder.KernelDownloadScript(realBuilder, nil, kr)
+			// Patch kernel download script to echo KERNELDIR.
+			// We need to capture KERNELDIR to later pass it as env variable to the build.
+			kernelDownloadScript += "\necho $KERNELDIR"
```

Comment:
`local` executor needs to fetch KERNELDIR after `exec.Command` runs.
The only way is to fallback at the old method of echoing `KERNELDIR` at the end of the script; then we will parse script stdout line by line and the last one will be our KERNELDIR path.